sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb

```diff
@@ -0,0 +1,989 @@
+{
```

Comment:
<p>Can you add a little detail around how you are selecting this model version? Are they sorted by ascending latency by default? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
Done.

Reply:
I added details about the sorting and how I choose the model with the lowest latency real-time inference.